hotfixes/wc-api-dev-email-order-url.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Provides a hotfix that points the admin order URL to WordPress.com instead of wp-admin.
+ *
+ * @since 0.8.2
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_api_dev_email_get_wpcom_order_link( $order_id ) {
+	$site_slug = preg_replace( '/^https?:\/\//', '', get_site_url() );
```

Comment:
Probably not something to worry about immediately, but this won't work for URLs where WP is in a subfolder. You can steal Jetpack's [`build_raw_urls` function](https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L6371) though (and it looks like this is usually passed `get_home_url`, not `get_site_url`).